usefulUtilities/hiFiCalendar/entityScripts/showClickForTheCalendarAppClientScript.js

```diff
@@ -0,0 +1,95 @@
+//  showClickForTheCalendarAppClientScript.js
+//
+//  Created by Milad Nazeri on 7-1-2019
+//  Copyright 2019 High Fidelity, Inc.
+// 
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+
+(function() {
+
+    var _this = null;
+    function ShouldShowClickForTheCalendarApp() {
+        _this = this;
+        
+        _this.canGetAndSetPrivateUserData = false;

```

Comment:
why do you need this flag? seems like you don't need it and it adds extra state

Reply:
Agreed, removed.